### PR TITLE
[Keymap] Personal keymap for crkbd

### PR DESCRIPTION
**Pull request by Jpe230**
Personal keymap for crkbd

## Description
- Renders on slave an animation of a cat which speed varies based on WPM (similar to Bongo Cat but simpler)
- Implement "a SArCasM moDE" 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/crkbd/keymaps/jpe230/keymap.c [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+        oled_write_ln_P(logo[0][0], false);
+        oled_write_ln_P(logo[0][1], false);
+    } else {
+        oled_write_ln_P(logo[1][0], false);
+        oled_write_ln_P(logo[1][1], false);
+    }
+}
+
+// 'frame_0_delay-0', 128x32px
+const char PROGMEM pkmn_frame_0 [] PROGMEM = {
```
Comment (drashna) [inline keyboards/crkbd/keymaps/jpe230/keymap.c:161]: Given nintendo's recent warpath... including an image of pikachu is definitely a no-go.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/crkbd/keymaps/jpe230/keymap.c (hunk 1 above)
Comment (Jpe230) [reply keyboards/crkbd/keymaps/jpe230/keymap.c]: Sad to know that Pikachu is not going to be part of my keymap but I understand why.

**Review by drashna — APPROVED** [review]

File: keyboards/crkbd/keymaps/jpe230/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Jpe230's CRKBD Layout
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/jpe230/README.md:1]: Please rename this file to lowercase.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/crkbd/keymaps/jpe230/keymap.c [hunk 3]
```diff
@@ -0,0 +1,406 @@
...
+bool uppercase;
+uint8_t prev_upper;
+uint8_t prev_lower;
+
+void keyboard_post_init_kb(void) {
+    tog_sarcasm    = false;
+    uppercase      = false;
+    prev_upper     = 0;
+    prev_lower     = 0;
+}
```
Comment (drashna) [inline keyboards/crkbd/keymaps/jpe230/keymap.c:337]:
```suggestion
bool tog_sarcasm = false;
bool uppercase = false;
uint8_t prev_upper = 0;
uint8_t prev_lower = 0;
```

**Review by fauxpark — APPROVED** [review]

**Pull request merged**